### PR TITLE
docs: update readme, fix formatting, clearer structure

### PR DESCRIPTION
**Pull request by beejjacobs**
## Describe your changes

The README has been updated to fix some formatting issues, have a clearer structure, and update out of date instructions.

## Issue ticket number and link

N/A

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,185 +1,217 @@
...
+* [Docker](https://www.docker.com/products/docker-desktop) (to run a local db)
 
-## Developing
+### Database Setup
 
-### Built With
+#### Database Server Setup
 
-The entire solution is built on top [Express.js](https://expressjs.com/) and [Nunjucks templating language](https://mozilla.github.io/nunjucks/templating.html), both for the web and api backends.
+Use Docker to run an instance of SQL Server, replace "<YourStrong@Passw0rd>":
```
Comment (fabfab-kc) [inline README.md:25]: Need to ensure the password complexity for SQL server is adequate, on some versions may lead to a misleading "login failed" error.

File: README.md [hunk 2]
```diff
@@ -1,185 +1,217 @@
...
 
-Before you get started you need to make sure you are running the latest Node LTS version and latest NPM version.
+**Notes for M1 Macs**
 
-### Setting up Dev
+For M1 Macs until [this issue](https://github.com/microsoft/mssql-docker/issues/668) gets resolved you should use the following instructions, or as an alternative you can use [colima](https://github.com/abiosoft/colima) instead of Docker.
 
-Here's a brief intro about what a developer must do in order to start developing
-the project further:
+Additionally, SQL Client tools are not included in the sql-edge image for ARM64, so you won't be able to use sqlcmd from with the container. One workaround for this is to use the mcr.microsoft.com/mssql-tools docker image. You can run this temporarily in another container and connect to your `pins_sql_server` container by name if they are on the same network. Firstly, create a network:
```
Comment (fabfab-kc) [inline README.md:35]: An alternative to running sql client docker images would be to run Azure Data Studio (https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall) and connect to the docker image

Comment (beejjacobs) [reply README.md:37]: Note added: https://github.com/Planning-Inspectorate/back-office/pull/874/commits/1e280801b6200916c0ca3eba880abac841920c12

File: README.md (hunk 1 above)
Comment (beejjacobs) [reply README.md:25]: Good point, note added: https://github.com/Planning-Inspectorate/back-office/pull/874/commits/539bc62797a589e78375baa4f1e3e0ea7e6a13b9

**Pull request merged**